authentication/certificate-types-descriptions.adoc

```diff
@@ -63,6 +63,44 @@ to serve as a placeholder until you configure a custom default certificate. Do
 not use Operator-generated default certificates in production clusters.
 ====
 
+[discrete]
+== Workflow
+
+.Custom certificate workflow
+
+image::custom-workflow.png[custom ingress certificate workflow]
+
+
+.Default certificate workflow
+
+image::default-workflow-4-5.png[default ingress certificate workflow]
+
+image:darkcircle-0.png[20,20] An empty `defaultCertificate` field causes the Ingress Operator to use its self-signed CA to generate a serving certificate for the specified domain.
+
+image:darkcircle-1.png[20,20] The default CA certificate and key generated by the Ingress Operator. Used to sign Operator-generated default serving certificates.
+
+image:darkcircle-2.png[20,20] The wildcard default serving certificate, created by the Ingress Operator and signed using the generated default CA certificate.
+
+image:darkcircle-3.png[20,20] The router deployment. Uses the certificate in `secrets/router-certs-default` as its default front-end server certificate.
+
+image:darkcircle-4.png[20,20] The contents of the wildcard default serving certificate (public and private parts) are copied here to enable OAuth integration.
+
+image:darkcircle-5.png[20,20] The `configmaps/router-ca` object is removed in {product-title} {product-version}.
+
+image:darkcircle-6.png[20,20] The public (certificate) part of the default serving certificate. Replaces the `configmaps/router-ca` resource.
+
+image:darkcircle-7.png[20,20] The user updates the cluster proxy configuration with the CA certificate that signed the `ingresscontroller` serving certificate. This enables components like `auth`, `console`, and the registry to trust the serving certificate.
+
+image:darkcircle-8.png[20,20] The cluster-wide trusted CA bundle containing the combined {op-system-first} and user-provided CA bundles or an {op-system}-only bundle if a user bundle is not provided.
+
+image:darkcircle-9.png[20,20] The custom CA certificate which instructs other components (for example, `auth` and `console`) to trust an `ingresscontroller` configured with a custom certificate.
```

Comment:
s/custom CA certificate/custom CA certificate bundle/. I say `bundle` because the configmap may contain more than 1 trusted CA cert.